esphome/components/ble_presence/binary_sensor.py

```diff
@@ -37,6 +38,7 @@ def _validate(config):
             cv.Optional(CONF_IBEACON_MAJOR): cv.uint16_t,
             cv.Optional(CONF_IBEACON_MINOR): cv.uint16_t,
             cv.Optional(CONF_IBEACON_UUID): cv.uuid,
+            cv.Optional(CONF_MIN_RSSI): cv.uint16_t,
```

Comment:
The rssi reported is negative and should be using this:
```suggestion
            cv.Optional(CONF_MIN_RSSI): cv.decibel,
```